spec/features/snack_show_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+describe "snack show page" do
+  it "displays name, price" do
+    owner = Owner.create(name: "Sam's Snacks")
+    turing = owner.machines.create(location: "Turing Basement")
+    dons  = owner.machines.create(location: "Don's Mixed Drinks")
+    kings  = owner.machines.create(location: "King Sooper")
+    snack_1 = Snack.create(name: "KitKat", price: 1)
+    snack_2 = Snack.create(name: "Oreo", price: 1.5)
+    snack_3 = Snack.create(name: "Chips", price: 2)
+    snack_4 = Snack.create(name: "Golden Chocolate", price: 49)
+
+    dons.add_snack(snack_3)
```

Comment:
To address what you mentioned in your comment, you do not need to make this add snack method. You can do this using AR right here in your test. My go to is by creating a MachineSnack. Might be something like this: 

```ruby 
MachineSnack.create(machine: dons, snack: snack_3)
MachineSnack.create(machine: dons, snack: snack_2)
```
...etc...

Reply:
Yeah, initially I had both `dons.snacks.create(...)` and `MachineSnack.create(machine: ..., snack: ...)` but then when I checked `dons.snacks` - it would duplicate the snack - so I had to get rid of one. It was towards the end I thought to instantiate them separately `dons = sam.machines.create` and `snack_1 = Snack.create` and I knew that I had to link them together, but in the moment, my initial thought was just to shovel them together.